docs/integrations/web-servers/opentelemetry/squid-proxy-opentelemetry.md

```diff
@@ -0,0 +1,150 @@
+---
+id: squid-proxy-opentelemetry
+title: Squid Proxy - OpenTelemetry Collector
+sidebar_label: Squid Proxy - OTel Collector
+description: Learn about the Sumo Logic OpenTelemetry App for Squid Proxy.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/web-servers/squid-proxy.png')} alt="Thumbnail icon" width="75"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The [Squid](http://www.squid-cache.org/Intro/) Proxy app is a logs app that helps you monitor activity in Squid Proxy. The preconfigured dashboards provide insight into served and denied requests; HTTP response codes; URLs experiencing redirects, client errors, and server errors; and quality of service data that helps you understand your users' experience.
+
+Squid logs are sent to Sumo Logic through OpenTtelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Squid-Proxy-OpenTelemetry/Squid-Proxy-Schematics.png')} alt="Schematics" />
+
+## Prerequisites
+
+By default, the squid proxy will write the access log to the log directory that was configured during installation. For example, on Linux, the default log directory is /var/log/squid/access.log. If the access log is disabled then you must enable the access log by following these [instructions](https://wiki.squid-cache.org/SquidFaq/SquidLogs).
+
+## Fields creation in Sumo Logic for Squid
+
+Following are the [fields](https://help.sumologic.com/docs/manage/fields/) which will be created as part of Squid App install if not already present:
+
+- **`webengine.cluster.name`** - User configured. Enter a name to identify this Squid cluster. This cluster name will be shown in the Sumo Logic dashboards.
+- **`webengine.system`**  - Has a fixed value of **squidproxy**.
+- **`sumo.datasource`** - Has a fixed value of **squidproxy**.
+
+## Collection Configuration & App installation
+
+As part of setting up the collection process and app installation user can select the App from App Catalog and follow the steps below:
+
+### Step 1: Set up Collector
+
+:::note
+If you want to use an existing OpenTelemetry Collector, you can skip this step by selecting the "Use an existing Collector" option.
+:::
+
+If you want to use an existing OTel Collector then this step can be skipped by selecting the option of using an existing Collector.
```

Comment:
This line is already covered as part of the note above it. This line can be deleted.